### PR TITLE
add active participation component

### PR DESCRIPTION
**Pull request by kortschak**
@jimmybgammyknee @steveped Please take a look when time permits.

### CODE REVIEW

**Comment by jimmybgammyknee** [comment]
Can you elaborate on that a bit? What do you mean by "issues filed against found in practicals"? Are you going to actually add errors so that they can create issues associated to them?

**Comment by kortschak** [comment]
I can generate errors without trying. I know some are there. Clarifications of pracs, spelling etc always persist. I'm not requiring that they do this, but it would be counted.

**Comment by jimmybgammyknee** [comment]
Well its a good way to make sure that they're engaged. Seems fine to me

File: README.md [hunk 1]
```diff
@@ -79,6 +80,11 @@ Have you:
 	- Emails should have a meaningful subject.
 - [ ] Handed in the assignment before the due time (Friday tutorial 9am)?
 
+#### Active Participation (3%)
+
+A small component of the course assessment will be for active participation in practicals and tutorials.
+Since the majority of the course material is open source, issues filed against found in practicals, tutorials and lectures, and contributions to fixing those errors will count to this component.
```
Comment (kortschak) [inline README.md]: See, I wasn't even trying: "issues filed against found in practicals". Missing word.

**Pull request merged**